series-tvdb.en.yaml

```diff
@@ -659,3 +659,69 @@ entries:
     seasons:
       - season: 1
         anilist-id: 20966
+        
+    - title: "FLCL"
+    synonyms:
+      - "FLCL Alternative"
+      - "FLCL Progressive"
+    seasons:
+      - season: 1
+        anilist-id: 227
+      - season: 2
+        anilist-id: 21748
+      - season: 3
+        anilist-id: 21746
+        
+  - title: "I Tried Asking While Kowtowing"
+    synonyms:
```

Comment:
Japanese synonym? `Dogeza de Tanondemita` ? And what's with the `<DOGEZA>` prefix?